projects/configs/longitudinal-dynamics-of-mri-targeted-prostate-cancer.yaml

```diff
@@ -0,0 +1,34 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "longitudinal-dynamics-of-mri-targeted-prostate-cancer"
+    modalities: ["MR"]
+
+tag_operation_files:
+    base:
+        - "base.yaml" #Expected base config file for any project
+        - "mri.yaml"
+        - "diffusion-weighted-mri.yaml"
+    manufacturer_overrides: ["mri.yaml", "mri-diffusion.yaml"]
+
+allowed_manufacturers:
+  - regex: ".*"
+    exclude_series_numbers: []
+
+min_instances_per_series: 1
```

Comment:
can we use the default of 2, or will some series have a single instance?

Reply:
Oh yeah, should be 2 for sure